content/en/docs/tasks/inject-data-application/define-command-argument-container.md

```diff
@@ -46,37 +46,47 @@ file for the Pod defines a command and two arguments:
 
 1. Create a Pod based on the YAML configuration file:
 
-        kubectl create -f https://k8s.io/examples/pods/commands.yaml
+    ```shell
+    kubectl create -f https://k8s.io/examples/pods/commands.yaml
+    ```
 
 1. List the running Pods:
 
-        kubectl get pods
+    ```shell
+    kubectl get pods
+    ```
 
     The output shows that the container that ran in the command-demo Pod has
     completed.
 
 1. To see the output of the command that ran in the container, view the logs
 from the Pod:
 
-        kubectl logs command-demo
+    ```shell
+    kubectl logs command-demo
+    ```
 
     The output shows the values of the HOSTNAME and KUBERNETES_PORT environment
     variables:
 
-        command-demo
-        tcp://10.3.240.1:443
+    ```log
```

Comment:
I think `log` would not be recognized.

Reply:
I checked and `log` indeed is not recognized as a valid keyword for syntax highlighting. I've removed this keyword and changed it to \`\`\`.